iec61850serversimulator/src/main/java/com/cgi/iec61850serversimulator/Relay.java

```diff
@@ -59,8 +61,12 @@ public void setSchedules(Schedule[] schedules) {
 	}
 
 	public void initializeSchedules(ModelNode scheduleInfo, int relayNr) {
+		String SWITCH_ROOT = "SWDeviceGenericIO/XSWC";
```

Comment:
Move this to the top of the class and make it "public static final String ...", to mark it as a constant.

Reply:
Done